packages/app/src/specs/spec-utils.ts

```diff
@@ -2,127 +2,7 @@ import fuzzySort from 'fuzzysort'
 import type { FoundSpec } from '@packages/types'
 import { ComputedRef, Ref, ref, watch } from 'vue'
 import _ from 'lodash'
-import { getRunnerConfigFromWindow } from '../runner'
-
-// Platform may not be available when this file is loaded (e.g. during component our component tests) so
-// we defer loading it until it's used
-let platform
-const getPlatform = (): string => {
-  if (!platform) {
-    platform = getRunnerConfigFromWindow().platform
-  }
-
-  return platform
-}
-const getRegexSeparator = () => getPlatform() === 'win32' ? /\\/ : /\//
-const getSeparator = () => getPlatform() === 'win32' ? '\\' : '/'
-
-export type FuzzyFoundSpec<T = FoundSpec> = T & {
-  fuzzyIndexes: {
-    relative: number[]
-    baseName: number[]
-  }
-}
-
-export type SpecTreeNode<T extends FoundSpec = FoundSpec> = {
-  id: string
-  name: string
-  children: SpecTreeNode<T>[]
-  isLeaf: boolean
-  parent?: SpecTreeNode<T>
-  data?: T
-  highlightIndexes: number[]
-}
-
-export function buildSpecTree<T extends FoundSpec> (specs: FoundSpec[], root: SpecTreeNode<T> = { name: '', isLeaf: false, children: [], id: '', highlightIndexes: [] }): SpecTreeNode<T> {
```

Comment:
`spec-utils` ended up a dumping ground for random things - all of these functions have one very specific purpose, which is to build the spec tree, so I moved them to a more accurately named file.